tests/unit/drop-zone.spec.js

```diff
@@ -64,23 +64,32 @@ describe('DropZone', () => {
       { args: [], name: 'entered' },
     ]);
 
-    w.vm.client.addFile({ name: 'file1', data: '' });
+    const now = Date.now();
+
+    // Freeze time
+    const RealDate = Date;
+    global.Date = jest.fn(
+      (...args) => (args.length ? new RealDate(...args) : new RealDate(now)),
+    );
+
+    const file = new File([''], 'file1', { type: 'text' });
+    w.vm.client.addFile(file);
+
+    // Unfreeze time
+    global.Date = RealDate;
 
     // Check the drop event
     w.trigger('drop', { dataTransfer: { files: [] } });
 
     const inputEvent = [
       {
-        data: {
-          data: '',
-          name: 'file1',
-        },
+        data: file,
         extension: '',
-        id: 'uppy-file1',
+        id: `uppy-file1-text-${now}`,
         isRemote: false,
         meta: {
           name: 'file1',
-          type: 'application/octet-stream',
+          type: 'text',
```

Comment:
Why does this change to text?

Reply:
Becaus of this line: `const file = new File([''], 'file1', { type: 'text' });`. If I don't provide a `type` it will do something even weirder.

Reply:
👍